test/configCases/library/umd/webpack.config.js

```diff
@@ -1,10 +1,22 @@
-module.exports = {
-	output: {
-		libraryTarget: "umd",
-		library: {
-			root: "testLibrary",
-			amd: "test-library",
-			commonjs: "test-library"
+module.exports = [
```

Comment:
I am not sure if it is fine to edit an existing test.

Reply:
They are quite similar so I thought they can live in the same folder. But it's not a big deal I can create a separate case. Interesting that I did exactly that in the first place but then I thought why not together? 😃